test/modules.js

```diff
@@ -0,0 +1,38 @@
+const test = require('tape')
+const combine = require('depject')
+const bulk = require('bulk-require')
+const path = require('path')
+
+process.env.ssb_appname = 'test'
+
+console.log(path.join( __dirname, '../modules_core'))
+
+
+console.log(`
+/////////
+// CURRENTLY TEST BROKEN
+//
+// It's bulk-require. I've tried using bulkify but it it does not seem to play well with brfs
```

Comment:
as i said before, `bulkify` and `brfs` play fine together. what order are they being run in? we want `bulkify` to be first since it adds new modules. but looking at the package.json you don't have `bulkify` at all there.